### ISSUE TITLE
Add contributor credit standard to the contributing guide

### ISSUE THREAD

**Issue by theekrystallee** [hiero-ledger/hiero-docs#559]
## 🆕🐥 First Timers Only
This issue is reserved for people who have never contributed to [Hedera](https://hedera.com) or any open source project in general.
We know that creating a pull request (PR) is a major barrier for new contributors.
The goal of this issue and all other issues labeled by [**'good first issue'**](https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Ahashgraph+archived%3Afalse+label%3A%22good+first+issue%22+) is to help you make your first contribution to Hedera.

## 👾 Description of the issue
We don't have a standardized way of recognizing contributions made to docs by outside contributors. Recognizing contributors publicly will help incentivize and encourage a collaborative community.

### Proposed solution

Update the Hedera Docs [Contributing Guide](https://docs.hedera.com/hedera/support-and-community/contributing-guide) to make it easier for contributors to receive credit for their work.

**1. Add a new "Contributor Credit" section:**
- Place this section directly below [Submit Demo Applications](https://docs.hedera.com/hedera/support-and-community/contributing-guide#submit-demo-applications).
- Clearly outline the steps for contributors to add their credit for contributions. Steps are outlined below, but feel free to expand them with helpful tips for clarity.
- Update the "[Create Pull Requests](https://docs.hedera.com/hedera/support-and-community/contributing-guide#create-pull-requests)" section of the contributing guide to include a point on adding contributor credit and link to the new "Contributor Credit" section.

**2. Add a section for Contributor Credit Format:**
- Include a new subsection in the [Style Guide](https://docs.hedera.com/hedera/support-and-community/contributing-guide#style-guide) with guidelines on the format for contributor credit (see the example credit below).

### Steps for Contributors (to be outlined in the "Contributor Credit" section mentioned above):
**1. Add Contributor Credit:**
- Include your contributor credit at the bottom of the docs page you are contributing to. See the live example [here](https://docs.hedera.com/hedera/getting-started/environment-set-up#code-check) for reference.

**2. Use Your Name or GitHub Username:**
- Either your first name or your GitHub username is acceptable. Hyperlink it to your GitHub profile.

**3. Include in the PR:**
- Ensure the credit is part of your pull request, along with your content contribution. This will allow you to be acknowledged on the live documentation once the PR is merged.

### Example Contributor Credit:

> **Contributors**: [krystal](https://github.com/theekrystallee)
> or
> **Contributors**: [@theekrystallee](https://github.com/theekrystallee)

The first example of contributor credit is live and can be found at the end of the [Code Check section](https://docs.hedera.com/hedera/getting-started/environment-set-up#code-check). Use this as a guide when adding your own credit to a page.

When submitting a PR, make sure to follow these steps to ensure you receive proper credit for your contributions.

### Additional Tips for First-Time Contributors:
- Ensure that your credit follows the exact format specified in the style guide to maintain consistency.
- Double-check the placement of your credit at the bottom of the page.
- If you’re unsure how to add hyperlinks in Markdown, refer to the [Markdown Guide](https://www.markdownguide.org/).
- Review your PR for any formatting issues to prevent delays in merging.

## 📋 Step by step guide to do a contribution
If you have never contributed to an open source project at GitHub, the following step-by-step guide will introduce you to the workflow. More information and concrete samples for shell commands for each step can be found in our [CONTRIBUTING.md](https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md) file.
A more detailed general documentation of the GitHub PR workflow can be found [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).

- [ ] **Claim this issue:** Comment below that you are interested in working on the issue
- [ ] **Wait for assignment:** A community member with the given rights will add you as an assignee of the issue
- [ ] **Fork the repository:** You can do that in GitHub (by simply clicking the 'fork' button).
- [ ] **Check out the forked repository**
- [ ] **Create a feature branch** for the issue. We do not have a hard naming definition for branches but it is best practice to prefix the branch name with the issue id.
- [ ] **Solve the issue** in your branch.
- [ ] **Commit your changes:** Here, it is needed to add `sign-off` information to the commit to accept the "Developer Certificate of Origin" (https://developercertificate.org). More details can be found in our [CONTRIBUTING.md](https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md)
- [ ] **Start a Pull Request (PR)**: We have a pattern for naming pull requests that a GitHub Action checks. We use that pattern to support the creation of automatic release notes.
- [ ] **Check GitHub Actions:** Several GitHub Actions will be triggered automatically for each PR. If a GitHub Action fails and you do not understand the cause of that error do not hesitate to add a comment to the PR and ask the Hedera developer community for support.
- [ ] **Wait for reviews:** Members of the Hedera developer community will review your PR. If a reviewer finds any missing pieces or a problem, he or she will start a discussion with you and describe the next steps for solving the problem.
- [ ] **You did it 🎉:** We will merge the fix in the develop branch. Thanks for being part of the Hedera community as an open-source contributor ❤️

## 🎉 Contribute to Hacktoberfest

Solve this issue as part of the [Hacktoberfest](https://hacktoberfest.digitalocean.com) event and get a chance to receive cool goodies like a T-Shirt. 🎽

## 🤔 Additional Information

If you have any questions, just ask us directly in this issue by adding a comment. You can join our community chat at [Discord](https://hedera.com/discord). A general manual about open-source contributions can be found [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).

**Comment by bguiz** [comment]
Is this referencing the author/ editor cards that appear at the bottom of tutorial pages?

**Comment by theekrystallee** [comment]
No, this is just referencing credit for pages other than tutorials. The cards will remain for tutorials.
